tests/FakeItEasy.Specs/SelfInitializedFakesSpecs.cs

```diff
@@ -2,16 +2,26 @@ namespace FakeItEasy.Specs
 {
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
     using System.IO;
     using System.Linq;
     using FakeItEasy.SelfInitializedFakes;
+    using FakeItEasy.Tests.TestHelpers;
     using FluentAssertions;
     using Xbehave;
     using Xunit;
 
     public interface ILibraryService
     {
         int GetCount(string internationalStandardBookNumber);
+
+        string GetTitle(string internationalStandardBookNumber);
+
+        [SuppressMessage("Microsoft.Design", "CA1021:AvoidOutParameters", MessageId = "0#", Justification = "Required for test scenario")]
+        [SuppressMessage("Microsoft.Design", "CA1045:DoNotPassTypesByReference", MessageId = "1#", Justification = "Required for test scenario")]
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "ref", Justification = "There are no other consumers, so the name is safe.")]
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "out", Justification = "There are no other consumers, so the name is safe.")]

```

Comment:
I think we really need to disable these warnings in test projects... They don't make sense in this context, and the suppressions just clutter the code.